src/libnnstreamer-edge/nnstreamer-edge-data.c

```diff
@@ -542,7 +542,7 @@ nns_edge_data_deserialize (nns_edge_data_h data_h, void *data)
     ptr += ed->data[n].data_len;
   }
 
-  ret = nns_edge_metadata_deserialize (ed->metadata, ptr, meta_len);
+  ret = nns_edge_metadata_deserialize (ed->metadata, ptr, (size_t) meta_len);
```

Comment:
how about changing size_t to uint32_t in nns-edge repo?

Reply:
Updated. thanks.